common/presenters/allocation-to-summary-list-component.js

```diff
@@ -0,0 +1,40 @@
+const i18n = require('../../config/i18n')
+const filters = require('../../config/nunjucks/filters')
+
+function allocationToSummaryListComponent(allocation) {
+  const {
+    moves_count: movesCount,
```

Comment:
Can we use `moves.length` here too please so that we don't introduce more debt around `moves_count`.